### ISSUE TITLE
DeprecationWarning from distutils use for getting version

### ISSUE THREAD

**Issue by alexander-held** [scikit-hep/uproot4#552]
Hi, I ran into this deprecation warning in tests and wanted to report it. This is not really a "bug", but that category was the closest I could think of.

```bash
$ python --version
Python 3.9.10
$ python -Wall
Python 3.9.10 | packaged by conda-forge | (main, Jan 30 2022, 18:02:27)
[Clang 11.1.0 ] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import uproot
>>> uproot.__version__
'4.1.9'
>>> uproot.extras.awkward()
[...]/lib/python3.9/site-packages/awkward/__init__.py:14: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
  if distutils.version.LooseVersion(numpy.__version__) < distutils.version.LooseVersion(
[...]/lib/python3.9/site-packages/awkward/__init__.py:14: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
  if distutils.version.LooseVersion(numpy.__version__) < distutils.version.LooseVersion(
[...]/lib/python3.9/site-packages/uproot/extras.py:36: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
  if LooseVersion("1") < LooseVersion(awkward.__version__) < LooseVersion("2"):
[...]/lib/python3.9/site-packages/uproot/extras.py:36: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
  if LooseVersion("1") < LooseVersion(awkward.__version__) < LooseVersion("2"):
[...]/lib/python3.9/site-packages/uproot/extras.py:36: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
  if LooseVersion("1") < LooseVersion(awkward.__version__) < LooseVersion("2"):
<module 'awkward' from '[...]/lib/python3.9/site-packages/awkward/__init__.py'>
```

I did not explicitly test master, but looking at the code the behavior should be the same: https://github.com/scikit-hep/uproot4/blob/6fd8863d932ca52f45d84cf5eec5b9c81b1bcfec/src/uproot/extras.py#L35

**Comment by alexander-held** [comment]
While looking at `awkward` I ran into https://github.com/scikit-hep/awkward-1.0/issues/1185 and https://github.com/scikit-hep/awkward-1.0/pull/1196, so it looks to already be changed there (I see the same warning in `1.7.0`, but the change looks like it came after that release).

**Comment by alexander-held** [comment]
I might be misunderstanding the setup, but I think this still exists in `uproot`. On master I see `distutils` used: https://github.com/scikit-hep/uproot4/blob/6fd8863d932ca52f45d84cf5eec5b9c81b1bcfec/src/uproot/extras.py#L14

The warnings above also include those coming from `awkward`, but I think an update of `awkward` will only make some go away. I started seeing this when switching from Python 3.8 to Python 3.9.

**Comment by jpivarski** [comment]
Yes, it still exists in Uproot. Oh! This is an Uproot issue! Reopening now.

**Comment by alexander-held** [comment]
Sorry for the confusion! Indeed, with `awkward` showing up a lot here, and me linking to it, that was not very clear.

**Comment by jpivarski** [comment]
Fixed by #564.
